### ISSUE TITLE
build from source fails

### ISSUE THREAD

**Issue by spvkgn** [jarun/nnn#144]
Can't build 2.1 on Ubuntu 18.04 x86_64
```
# apt policy libncurses5-dev
libncurses5-dev:
  Installed: 6.1-1ubuntu1
  Candidate: 6.1-1ubuntu1
  Version table:
 *** 6.1-1ubuntu1 500
        500 http://mirror.yandex.ru/ubuntu bionic/main amd64 Packages
        100 /var/lib/dpkg/status
# make
cc  -O3 -Wall -Wextra -Wno-unused-parameter  -o nnn src/nnn.c -lncurses
src/nnn.c: In function 'filterentries':
src/nnn.c:1116:14: warning: implicit declaration of function 'get_wch'; did you mean 'getch'? [-Wimplicit-function-declaration]
  while ((r = get_wch(ch)) != ERR) {
              ^~~~~~~
              getch
src/nnn.c: In function 'xreadline':
src/nnn.c:1254:3: warning: implicit declaration of function 'mvaddnwstr'; did you mean 'mvaddnstr'? [-Wimplicit-function-declaration]
   mvaddnwstr(y, x, buf, len + 1);
   ^~~~~~~~~~
   mvaddnstr
/tmp/ccneQDFu.o: In function `xreadline':
nnn.c:(.text+0x15c2): undefined reference to `mvaddnwstr'
nnn.c:(.text+0x15ec): undefined reference to `get_wch'
/tmp/ccneQDFu.o: In function `browse':
nnn.c:(.text+0x491b): undefined reference to `get_wch'
collect2: error: ld returned 1 exit status
Makefile:28: recipe for target 'nnn' failed
make: *** [nnn] Error 1
```

**Comment by jarun** [comment]
You need libncursesw5-dev

**Comment by krishnakumarg1984** [comment]
@jarun that's not strictly true. I could compile from source with just the `libncursesw10` package.

**Comment by jarun** [comment]
`libncursesw10` is not available on Ubuntu 18.04 default repos.

**Comment by krishnakumarg1984** [comment]
@jarun Ah. Curiously, `libncursesw5-dev` is not available in cygwin repos. Do you think it is a good idea for you to consider making a note this information in the `README.md` for the project?

**Comment by jarun** [comment]
No I don't think I would have to document deps for platforms those are not officially certified.

**Comment by krishnakumarg1984** [comment]
@jarun  No. I meant making a note of the fact either `libncursesw10` or `libncursesw5-dev` shall suffice. I know it's not available in Ubuntu 18.04, but it is certainly available from arch, void, LFS and others.

**Comment by jarun** [comment]
https://github.com/jarun/nnn#dependencies
